app.rb

```diff
@@ -2,11 +2,21 @@
 require './student'
 require './teacher'
 require './book'
+require './writer'
+require 'json'
 
 class App
+  attr_reader :books, :people, :rentals
+
   def initialize
-    @books = []
-    @people = []
+    # read from json
+    @books = Writter.morph(JSON.parse(File.read('books.json')), 'books.json')
+    @people = Writter.morph(JSON.parse(File.read('people.json')), 'people.json')
+    @rentals = Writter.morph(JSON.parse(File.read('rentals.json')), 'rentals.json')
```

Comment:
- [ ] When I remove all the storage files and run your app I get a no file error. Kindly ensure that scenarios, where the JSON files do not exist, are handled properly - you can create the files before continuing with the options list, or at the moment when the data should be stored you should create the file but there should be no errors when files do not exist.

  ![image](https://user-images.githubusercontent.com/81468675/225297064-f12ef286-883d-4dc2-b4d0-2c1aa1d7fe94.png)